enterprise/frontend/src/embedding-sdk/components/public/StaticQuestion/StaticQuestion.tsx

```diff
@@ -65,18 +66,16 @@ const _StaticQuestion = ({
           card,
           result,
           loading: false,
-          cardError: null,
-          resultError: null,
+          error: null,
         }));
       })
-      .catch(([cardError, resultError]) => {
+      .catch(error => {
```

Comment:
To quote [this stackoverflow answer](https://stackoverflow.com/a/30378082)

> Promise.all is all or nothing. It resolves once all promises in the array resolve, or reject as soon as one of them rejects. In other words, it either resolves with an array of all resolved values, or rejects with a single error.


So we just need to show one error. We can then use the `SdkError` component here.